src/V8JsiRuntime_impl.h

```diff
@@ -114,8 +116,10 @@ while (!use_count_s_.compare_exchange_weak(current, current - 1))
   ;*/
 
     if (--use_count_s_ == 0) {
-      v8::V8::ShutdownPlatform();
-      platform_s_ = nullptr;
+      // We cannot shutdown the platform once created because V8 internally references bits of the platform from process-globals

```

Comment:
Do we need to change the API for this if it never dies? Could we track this somewhere?

Reply:
The public API didn't change, but overall this could use some cleanup.